sentry/src/lib.rs

```diff
@@ -7,20 +7,17 @@ use adapter::{prelude::*, Adapter};
 use chrono::Utc;
 use hyper::{Body, Method, Request, Response, StatusCode};
 use middleware::{
-    auth::{AuthRequired, Authenticate, IsAdmin},
+    auth::{AuthRequired, Authenticate},
     campaign::{CalledByCreator, CampaignLoad},
     channel::ChannelLoad,
     cors::{cors, Cors},
     Chain, Middleware,
 };
 use once_cell::sync::Lazy;
-use primitives::{
-    analytics::{query::AllowedKey, AuthenticateAs},
-    sentry::ValidationErrorResponse,
-    Config, ValidatorId,
-};
+use primitives::{sentry::ValidationErrorResponse, Config, ValidatorId};
 use redis::aio::MultiplexedConnection;
 use regex::Regex;
+use routes::routers::analytics_router;
```

Comment:
I haven't decided how exactly where we will put all `*_router` functions but it's good to remove them from the `lib.rs`.
These functions can later be removed, once we transition to a web framework